input/pagecontent/comparison.md

```diff
@@ -335,6 +373,106 @@ AU Core compliant resources are compliant with IPS requirements for Missing Data
 ##### Suppressed Data
 IPS does not include requirements for Suppressed Data.
 
+##### Additional Must Support elements
```

Comment:
Merge into the section Additional requirements.

Reply:
Done